src/mooc/main/tv/codely/mooc/video/domain/Video.java

```diff
@@ -20,4 +20,14 @@ public static Video publish(VideoTitle title, VideoDescription description) {
 
         return video;
     }
+
+    public static Video updateDescription(VideoTitle title, VideoDescription description) {
+        var video = new Video(title, description);
```

Comment:
In aggregates you can mutate the state, so instead of being this method static you could:
* First search for the `Video`
* then video.updateDescription(newDescription)